chains/neoxt4/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://xt4scan.ngd.network
```

Comment:
```suggestion
  - apiUrl: https://xt4scan.ngd.network/api
```

Reply:
Modified in dd41a59e930d91a34b54bb96f3fb58fcfc76b8c0.